R/get_ssm_by_regions.R

```diff
@@ -24,24 +26,31 @@
 #' @export
 #'
 #' @examples
-#' #basic usage, adding custom names from bundled ashm data frame
+#' # Create bed data by adding custom names from the bundled ashm data frame
 #' regions_bed = create_bed_data( GAMBLR.data::grch37_ashm_regions,
 #'                           fix_names = "concat",
 #'                           concat_cols = c("gene","region"),
 #'                           sep="-")
 #' 
 #' my_meta = get_gambl_metadata()
-#' # get a full MAF-format data frame for all aSHM regions on grch37 coordinates
+#' 
+#' # Get a full MAF-format data frame for all aSHM regions on grch37 coordinates
 #' ashm_maf = get_ssm_by_regions(regions_bed = regions_bed,
 #'                               these_samples_metadata = my_meta,
 #'                               streamlined = FALSE)
 #' 
-#' 
-#'
-#' one_region_maf = get_ssm_by_regions(regions_list = "2:136875000-136875097",
+#' # Using a vector of regions in the chr:start-end format
+#' one_region_maf = get_ssm_by_regions(regions_list = "chr2:136875000-136875097",
 #'                          streamlined = FALSE,
 #'                          projection = "grch37",
 #'                          these_samples_metadata = my_meta)
+#' 
```

Comment:
It would make more sense to show the user how to load a MAF from a tsv and how to make it into a maf_data object